scripts/zkai_minimal_transformer_block_benchmark_gate.py

```
@@ -0,0 +1,666 @@
+        writer.writerow({key: row[key] for key in TSV_COLUMNS})
+    return output.getvalue()
+
+
+def normalize_output_path(path: pathlib.Path) -> pathlib.Path:
+    try:
+        original_st = os.lstat(path)
+    except FileNotFoundError:
+        pass
+    else:
+        if stat.S_ISLNK(original_st.st_mode):
+            raise MinimalBlockBenchmarkError(f"refusing to write through symlink: {path}")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)